dawe_vocabs/settings.py

```diff
@@ -27,441 +27,1450 @@
 # Look up tables to be generated.
 lut_configs: List[LUTSchema] = [
     LUTSchema(
-        "http://vocabs.paratoo.tern.org.au:1337/api/lut-tier-2-observation-methods",
-        "Observation method tier 2 codes",
-        "A collection of observation tier 2 types and its codes.",
-        "f1592e71-cc16-4b81-90c4-06b418a5a766",
-        "observation method tier 2",
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-fauna-age-classes",
+        label="Age class codes",
+        description="A collection of age class types and its codes.",
+        collection_uuid="0e2641c3-0d7e-4d94-8cd7-02c21d564630",
+        uuid_namespace="age class",
     ),
     LUTSchema(
-        "http://vocabs.paratoo.tern.org.au:1337/api/lut-fauna-vagina-conditions",
-        "Vaginal condition codes",
-        "A collection of vaginal condition types and its codes.",
-        "09de3b86-616e-49af-a34c-903cf7dec443",
-        "vaginal condition",
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-fauna-behaviours",
+        label="Animal behaviour codes",
+        description="A collection of animal behaviour types and its codes.",
+        collection_uuid="1301857c-4e02-4000-966b-a0d0ce60368f",
+        uuid_namespace="animal behaviour",
     ),
     LUTSchema(
-        "http://vocabs.paratoo.tern.org.au:1337/api/lut-voucher-types",
-        "Voucher type codes",
-        "A collection of voucher types and its codes.",
-        "a3333793-95bd-47fc-ba90-b8fc65a40456",
-        "voucher type",
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-fauna-pouch-conditions",
+        label="Animal pouch condition codes",
+        description="A collection of animal pouch condition types and its " "codes.",
```

Comment:
hi Edmond, is the extra " syntax error?

Reply:
I think that is due to black's auto-formatting behaviour. In any case, it's not a syntax error. Two strings together performs a concatenation into one string.